src/Component/ChatBox/ChatBox.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState, useRef } from "react";
+import styled from "styled-components";
+
+import { ChatBubble } from "../ChatBubble/ChatBubble";
+import { actor } from "../dummyData";
+
+export interface ChatBoxProps {
+  dummyData: actor[];
+}
+
+const ChatBoxWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 80vw;
+  max-height: 75vh;
+  margin: 0 auto;
+  padding: 0.5em;
+  gap: 1rem;
+  background-color: rgb(244, 245, 247);
+  border: 2px solid #000;
+`;
+
+const MessagesWrapper = styled.div`
+  flex: 1;
+  overflow-y: scroll;
+  scroll-behavior: smooth;
+
+  &::-webkit-scrollbar {
+    width: 0em;
+  }
+`;
+
+const InputWrapper = styled.div`
+  display: flex;
+  gap: 1rem;
+`;
+
+const MessageInput = styled.textarea`
+  flex: 1;
+  padding: 0.5rem 1rem;
+  border: 2px solid #000;
+`;
+
+const EnterButton = styled.button`
+  padding: 0.5rem 1.5rem;
+  border: 2px solid #000;
+  border-radius: 3px;
+`;
+
+export const ChatBox = (props: ChatBoxProps) => {
+  const { dummyData } = props;
+
+  const [message, setMessage] = useState<string>("");
```

Comment:
```suggestion
  const [message, setMessage] = useState<string>(null);
```

Reply:
it's a good pattern to use null instead of empty string

Reply:
Added value as null
```diff
+ const [message, setMessage] = useState<string | null>(null);
```

Got typescript error
```diff
- Type 'string | null' is not assignable to type 'string | number | readonly string[] | undefined'
```

I am using message variable to assign value in input field
```js
<MessageInput
          rows={1}
          placeholder="Enter Message"
          value={message}
          onChange={(e) => setMessage(e.target.value)}
        />
```

Should I assign it to undefined instead of null ?
```diff
+ const [message, setMessage] = useState<string | undefined>(undefined);
```

Reply:
Ignore my comments on this one, makes sense to have it as 
```
const [message, setMessage] = useState<string>("");
```

Reply:
Okay